README.md

```diff
@@ -1,4 +1,9 @@
 # NGI-NextflowDocs
+
+Project is likely to be out of date and might be incorrect in some case.
+
+We advised to have a look at [Nextflow documentation](https://www.nextflow.io/docs/latest/index.html) or at the [nf-core website](https://nf-core.github.io/), or to join the gitter channels for some direct contact with developers: [nextflow channel](https://gitter.im/nextflow-io/nextflow) or [nf-core Lobby](https://gitter.im/nf-core/Lobby).
```

Comment:
"We recommend"

Reply:
At least I should have wrote `we advise` ...